### ISSUE TITLE
Why comment out opt.load_state_dict(checkpoint['opt'])?

### ISSUE THREAD

**Issue by Peterande** [hustvl/LightningDiT#44]
Why did you comment out # opt.load_state_dict(checkpoint['opt'])?

**Comment by JingfengYao** [comment]
This was a minor unresolved issue during the experimental process at the time. Under the condition of only inheriting the model, the loss curve could still resume smoothly.
